idd/Energy+.idd.in

```diff
@@ -73581,17 +73581,30 @@ Chiller:Electric:EIR,
        \note PLR is the chilled water plant loop part load ratio (actual/design)
        \type object-list
        \object-list UnivariateFunctions
-  A20, \field Temperature Difference Across Condenser Schedule Name
+  A19, \field Temperature Difference Across Condenser Schedule Name
```

Comment:
Not sure why but the idd had a mismatch with alpha field numbers in several chiller models.